hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/upgrade/UpgradeActionRecon.java

```diff
@@ -42,15 +42,12 @@
  * Example usage:
  *
  * <pre>
- * {@code
- *
- * @UpgradeActionRecon(feature = FEATURE_NAME, type = FINALIZE)
- *  public class FeatureNameUpgradeAction implements ReconUpgradeAction {
- *     @Override
- *     public void execute() throws Exception {
- *       // Custom upgrade logic for FEATURE_1
- *     }
- *  }
+ * &#64;UpgradeActionRecon(feature = FEATURE_NAME, type = FINALIZE)
+ * public class FeatureNameUpgradeAction implements ReconUpgradeAction {
+ *   &#64;Override
+ *   public void execute() throws Exception {
+ *     // Custom upgrade logic for FEATURE_1
```

Comment:
Thanks a lot, @adoroszlai, for fixing this JavaDoc!

I have one small request: could you please rename "FEATURE_1" to "FEATURE_NAME"? I missed it last time. Thank you!

Reply:
I really don't want to run full CI just to fix this existing typo.  Please make a note of the problem and fix it when you revisit this file for other reasons.